source/Octostache/Templates/Functions/TextManipulationFunction.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Linq;
+using System.Text;
+
+namespace Octostache.Templates.Functions
+{
+    internal class TextManipulationFunction
+    {
+        public static string ToBase64(string argument, string[] options)
+        {
+            if (options.Any() || argument == null)
+            {
+                return null;
+            }
+            var argumentBytes = Encoding.UTF8.GetBytes(argument);
```

Comment:
We should probably support the user ending through an argument that can be `utf8` or `unicode` (default to `utf8` if missing)

Reply:
We can add that if someone requests it

Reply:
The problem is its not obvious to users until it breaks.. Ill add it